src/cascadia/TerminalApp/Pane.cpp

```diff
@@ -320,12 +320,12 @@ void Pane::_ControlConnectionStateChangedHandler(const TermControl& /*sender*/,
     }
 
     const auto& settings = CascadiaSettings::GetCurrentAppSettings();
-    auto paneProfile = settings.FindProfile(_profile.value());
+    auto& paneProfile = settings.FindProfile(_profile.value());

```

Comment:
prefer _not_ using `&` on this one, since it's a reference to a temporary return value from a function

Reply:
that's dangerous.